plasmapy/physics/tests/test_quantum.py

```diff
@@ -71,26 +61,29 @@ def test_thermal_deBroglie_wavelength():
     r"""Test the thermal_deBroglie_wavelength function in quantum.py."""
     lambda_dbTh = thermal_deBroglie_wavelength(T_e)
     # test a simple case for expected value
+    expectStr = "Thermal deBroglie wavelength is off expected result for 1eV"
```

Comment:
We could use the python 3.6 technique I mentioned in #152 here! Then again, pytest should show the value of `lambda_dbTh` within `np.isclose` so I guess that's not absolutely necessary.